package.json

```diff
@@ -47,7 +47,7 @@
     "react-dom": "^18.3.1",
     "react-google-recaptcha": "^3.1.0",
     "react-i18next": "^15.0.2",
-    "react-icons": "^5.2.1",
+    "react-icons": "^5.3.0",
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.